clients/src/main/java/org/apache/kafka/common/requests/ElectLeadersResponse.java

```diff
@@ -0,0 +1,155 @@
+        this.data = new ElectLeadersResponseData(struct, version);
+    }
+
+    public ElectLeadersResponse(Struct struct) {
+        this.version = (short) (ElectLeadersResponseData.SCHEMAS.length - 1);
+        this.data = new ElectLeadersResponseData(struct, version);
+    }
+
+    public ElectLeadersResponse(
+            int throttleTimeMs,
+            short errorCode,
+            Map<String, Map<Integer, ApiError>> electionResults) {
+        this(throttleTimeMs, errorCode, electionResults, (short) (ElectLeadersResponseData.SCHEMAS.length - 1));
+    }
+
+    public ElectLeadersResponse(
+            int throttleTimeMs,
+            short errorCode,
+            Map<String, Map<Integer, ApiError>> electionResults,
+            short version) {
+
+        this.version = version;
+        this.data = new ElectLeadersResponseData();
+
+        if (version >= 1) {
+            data.setErrorCode(errorCode);
+        }
+
+        for (Map.Entry<String, Map<Integer, ApiError>> topicEntry : electionResults.entrySet()) {
+            ReplicaElectionResult replicaElectionResult = new ReplicaElectionResult();
+            replicaElectionResult.setTopic(topicEntry.getKey());
+
+            for (Map.Entry<Integer, ApiError> partitionEntry : topicEntry.getValue().entrySet()) {
+                PartitionResult partitionResult = new PartitionResult();
+                partitionResult.setPartitionId(partitionEntry.getKey());
+                partitionResult.setErrorCode(partitionEntry.getValue().error().code());
+                partitionResult.setErrorMessage(partitionEntry.getValue().message());
+
+                replicaElectionResult.partitionResult().add(partitionResult);
+            }
+
+            data.replicaElectionResults().add(replicaElectionResult);
+        }
+    }
+
+    public ElectLeadersResponseData data() {
+        return data;
+    }
+
+    public short version() {
+        return version;
+    }
+
+    @Override
+    protected Struct toStruct(short version) {
+        return data.toStruct(version);
+    }
+
+    @Override
+    public int throttleTimeMs() {
+        return data.throttleTimeMs();
+    }
+
+    @Override
+    public Map<Errors, Integer> errorCounts() {
+        HashMap<Errors, Integer> counts = new HashMap<>();
+        for (ReplicaElectionResult result : data.replicaElectionResults()) {
+            for (PartitionResult partitionResult : result.partitionResult()) {
+                Errors error = Errors.forCode(partitionResult.errorCode());
+                counts.put(error, counts.getOrDefault(error, 0) + 1);
+            }
+        }
+        return counts;
+    }
+
+    public static ElectLeadersResponse parse(ByteBuffer buffer, short version) {
+        return new ElectLeadersResponse(
+                ApiKeys.ELECT_LEADERS.responseSchema(version).read(buffer), version);
+    }
+
+    @Override
+    public boolean shouldClientThrottle(short version) {
+        return version >= 3;
```

Comment:
Seems to be a bug from the original. I think we can just return `true` since client-side throttling was introduced before this API.